### PR TITLE
Problem: cli redundant migrated key log in stdout

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 - [#110](https://github.com/crypto-com/pystarport/pull/110) add event_query_tx_for to allow subscribe and wait for transaction.
 - [#112](https://github.com/crypto-com/pystarport/pull/112) add cmd for migrate keystore.
 - [#113](https://github.com/crypto-com/pystarport/pull/113) support ibc related cmd.
+- [#](https://github.com/crypto-com/pystarport/pull/) avoid cli redundunt migrated key log in stdout.
```
Comment (mmsqe) [inline CHANGELOG.md:20]: for more [info](https://github.com/cosmos/cosmos-sdk/pull/19371)
```suggestion
- [#115](https://github.com/crypto-com/pystarport/pull/115) avoid cli redundunt migrated key log in stdout.
```

**Review by yihuang — APPROVED** [review]

**Comment by yihuang** [comment]
Aren't all command has similar issue?

**Comment by mmsqe** [comment]
> Aren't all command has similar issue?

Mainly when do `gov_vote` some key migrate in the middle without do `migrate_keystore` 1st.

**Comment by yihuang** [comment]
> > Aren't all command has similar issue?
> 
> Mainly when do `gov_vote` some key migrate in the middle without do `migrate_keystore` 1st.

but all commands have the same issue, just `gov_vote` happens to run first in integration test?

**Pull request merged**